examples/api/lib/widgets/magnifier/cupertino_magnifier.0.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+
+/// Flutter code sample for [CupertinoMagnifier].
+
+void main() => runApp(const CupertinoMagnifierApp());
+
+class CupertinoMagnifierApp extends StatelessWidget {
+  const CupertinoMagnifierApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const CupertinoApp(
+      theme: CupertinoThemeData(brightness: Brightness.light),
+      home: CupertinoMagnifierExample(),
+    );
+  }
+}
+
+class CupertinoMagnifierExample extends StatefulWidget {
+  const CupertinoMagnifierExample({super.key});
+
+  @override
+  State<CupertinoMagnifierExample> createState() => _CupertinoMagnifierExampleState();
+}
+
+class _CupertinoMagnifierExampleState extends State<CupertinoMagnifierExample> {
+  static const double magnifierRadius = 50.0;
+  Offset dragGesturePosition = Offset.zero;
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      navigationBar: const CupertinoNavigationBar(
+        middle: Text('CupertinoMagnifier Sample'),
+      ),
+      child: Center(
+        child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
```

Comment:
From line 42 to 75 there is too much indentation.

Reply:
Hey @RamonFarizel ,

unfortunately auto format is not an option on the Flutter repo for the moment (it might be in 2025), but for the moment, according to the contribution guide, readability is key and can't be achieved with current autoformat tool (it will be soon 🤞).

In that particular case, from your IDE, you can just select lines 42 to 75 and hit shift+tab to remove one level of indentation.

> I'm having too many formatting issues. How can I improve on it?

Do you mean for this PR or for other flutter/flutter contributions?
For this PR, as far as I can tell this indentation issue is the only remaining formatting issue.
(On the test side, it looks like you used autoformat, I will suggest some small changes later to make things a little more readable).

Reply:
Good to know! Thanks for your answer! 
Yeah, I was using auto format. 
My question was focused on future contributions. I want to ensure that I don't repeat these mistakes in the future.